include/fastrtps/rtps/resources/TimedCallback.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
```

Comment:
I think a best location for this file would be `include/fastrtps/timedevent`

Reply:
Moved